zipline/examples/buy_and_hold.py

```diff
@@ -22,6 +23,12 @@ def initialize(context):
     context.has_ordered = False
     context.stocks = stocks
 
+    # Explicitly set the commission to the "old" value until we can
+    # rebuild example data.
+    # github.com/quantopian/zipline/blob/master/tests/resources/
+    # rebuild_example_data#L105
```

Comment:
TLDR: What you have here is good.

It's actually still possible to rebuild the example data (with a couple minor tweaks), I did so [here](https://github.com/quantopian/zipline/pull/1910) only about a month ago. We now get benchmark data from Google instead of Yahoo, as seen [here](https://github.com/quantopian/zipline/blob/master/zipline/data/benchmarks.py#L46-L51).

_However_, it appears that as of only a week or two ago, Google changed the URL from which they are serving their financial data, causing pandas datareader to break. As a result, after checking out your branch I was unable to rebuild the data like I was previously (without having to either (1) make our own pandas-datareader fork, or (2) make a bunch of extra changes to the tests and test dates; neither of which I want or think we need to do).

For more info see the following:
- https://github.com/pydata/pandas-datareader/issues/391
- https://stackoverflow.com/questions/46259528/datareader-google-finance-date-not-working/46356247#46356247
- https://stackoverflow.com/questions/46243360/google-api-changed-for-data-from-google-finance

All that being said, I think your solution of keeping the old parameters for now is best. When the pandas datareader issue is fixed though I think this may be worth revisiting.

@richafrank @FreddieV4 Is this breakage something we should be worried about anywhere else?

Reply:
Yea, I think we need to revisit our benchmark downloading, either fixing google, adding back yahoo (via pandas-datareader if it supports it), or both.

Reply:
Filed https://github.com/quantopian/zipline/issues/1965